integrationTests/multiShard/block/executingMiniblocksSc_test.go

```diff
@@ -333,6 +347,7 @@ func TestShouldProcessWithScTxsJoinNoCommitShouldProcessedByValidators(t *testin
 
 		syncBlock(t, nodes, idxProposers, round)
 		round = incrementAndPrintRound(round)
```

Comment:
Seems we have 2 incrementAndPrintRound functions one in this package and one in integrationTests.
Please remove the local one

Reply:
Yes, it has been removed by Beni in its branch which just has been merged in development, so now it disappeared :)